tests/python/driver/tvmc/test_autotuner.py

```diff
@@ -207,3 +209,27 @@ def test_autotune_pass_context(mock_pc, onnx_mnist, tmpdir_factory):
     # AutoTVM overrides the pass context later in the pipeline to disable AlterOpLayout
     assert mock_pc.call_count == 2
     assert mock_pc.call_args_list[0][1]["opt_level"] == 3
+
+
+def test_filter_tasks_valid():
```

Comment:
Nit: it would be helpful to parameterize these tests similar to below, so failures are reported separately in the CI log